INSTALLATION.md

```diff
@@ -179,3 +178,19 @@ body and title, and you are required at least one tag.
 And then click to "Save Post in Q&A"
 
 ![img/post.png](img/post.png)
+
+## Optional: Help Topics
+
+If you are running a development server, you might not care a lot about what's in the help.  If you are planning to deploy a server for actual use, however, note that the seeds have some placeholder text you'll want to edit.  We have provided starting points (to be edited) for the following topics:
+
+- Terms of service (TOS)  
+- Code of conduct (COC)  
+- Privacy policy  
+- Spam policy  
+- Global (network) FAQ
+
+The corresponding posts in db/seeds/posts have some places marked with "$EDIT" where you will probably want to insert URLs, email addresses, and the like.  We recommend reviewing all of the content in these topics.  There are two ways to edit these topics:
+
+- Edit the seed files before seeding your database.  

```

Comment:
Perhaps you can add that they should use `UPDATE_POSTS=true rails db:seed` if they want to include the help topics in the seeding. Or was the plan to change/alter/rename this functionality?

Reply:
Oh, I forgot about that.  I think there are three options, right?
1. Edit the post seeds after you clone the repo and before you start setting up qpixel.
2. Set up qpixel, which includes seeds, and then set UPDAT_POSTS and seed again (right?).
3. Set up qpixel and edit the help in the UI.

Reply:
Other than this item, I believe I've incorporated all of your suggestions.  Thanks for the review!

Reply:
You say 3 options, but its not quite 3 options. It's either 1+2 (change in files, then seed the posts) or 2+3 (seed with posts, then update in UI). Both are fine options, but 2+3 is easier to do if you are non-technical and 1+2 may be easier if you want large changes.

Regarding the second point, you can do it like you suggest, or you can immediately pass the flag, both will have the same end result. The UPDATE_POSTS just adds another task to the seeding, it still performs the normal seeding as well.

Reply:
Thanks.  Updated.